src/Hakyll/Links/Parser.hs

```diff
@@ -0,0 +1,36 @@
+module Hakyll.Links.Parser
+       ( parseShortcut
+       ) where
+
+import Data.Text (Text)
+import Data.Void (Void)
+import Text.Megaparsec (Parsec, anySingle, noneOf, optional, parse, some, (<|>))
+import Text.Megaparsec.Char (alphaNumChar, char)
+
+import qualified Data.Text as T
+
+
+type Parser = Parsec Void Text
+
+{- | Parse a shortcut link. Allowed formats:
+
+@
+\@name
+\@name:text
+\@name(option)
+\@name(option):text
```

Comment:
I think that maybe it's a good idea to have consistent naming in the documentation and in the implementation. If I'm right, than the current correspondence is the following:

| Docs | Code |
|--|--|
|name|shortcut|
|option|opt|
|text|text|

I just think that it can help maintaining and understanding this code better :slightly_smiling_face: